beta-src/src/components/ui/WDUI.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from "react";
+import { Box, useTheme } from "@mui/material";
+
+import WDPositionContainer from "./WDPositionContainer";
+import Position from "../../enums/Position";
+import WDInfoDisplay from "./WDInfoDisplay";
+import { useAppSelector } from "../../state/hooks";
+import { gameOverview } from "../../state/game/game-api-slice";
+import WDInfoPanel from "./WDInfoPanel";
+import Device from "../../enums/Device";
+import { CountryTableData } from "../../interfaces";
+import Country from "../../enums/Country";
+
+interface WDUIProps {
+  device: Device;
+}
+
+const WDUI: React.FC<WDUIProps> = function ({ device }): React.ReactElement {
+  const theme = useTheme();
+  const {
+    alternatives,
+    excusedMissedTurns,
+    members,
+    name,
+    pot,
+    season,
+    user,
+    year,
+  } = useAppSelector(gameOverview);
+  console.log({
+    alternatives,
+    name,
+    pot,
+    season,
+    year,
+  });
+
+  const countryMap = {
+    Russia: Country.RUSSIA,
+    Germany: Country.GERMANY,
+    Italy: Country.ITALY,
+    Austria: Country.AUSTRIA,
+    England: Country.ENGLAND,
+    France: Country.FRANCE,
+    Turkey: Country.TURKEY,
+  };
+
+  const abbrMap = {
+    Russia: "RUS",
+    Germany: "GER",
+    Italy: "ITA",
+    Austria: "AUS",
+    England: "ENG",
+    France: "FRA",
+    Turkey: "TUR",
+  };
+
+  const constructTableData = (member) => {
+    const memberCountry: Country = countryMap[member.country];
+    return {
+      ...member,
+      abbr: abbrMap[member.country],
+      color: theme.palette[memberCountry].main,
+      power: memberCountry,
+      votes: {
+        cancel: member.votes.includes("Cancel"),
+        draw: member.votes.includes("Draw"),
+        pause: member.votes.includes("Pause"),
```

Comment:
we should use the enum for includes?

Reply:
Good point. We changed that enum to be all lowercase, but I can probably do something like:
```
const vote = Vote[Vote.cancel];
vote.charAt(0).toUpperCase() + vote.slice(1);
```

Probably add a utils for the uppercasing since we'll be using it I'm sure.